src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -1,652 +1,652 @@
-            connection.getPoolBagEntry().evicted = (config.getMinimumIdle() == 0);
-            connection.close();
-         }
-         catch (Throwable e) {
-            try {
-               shutdown();
-            }
-            catch (Throwable ex) {
-               e.addSuppressed(ex);
-            }
-
-            throw new PoolInitializationException(e);
-         }
-      }
-
-      fillPool();
-   }
-
-   private PoolStats getPoolStats()
-   {
-      return new PoolStats(TimeUnit.SECONDS.toMillis(1)) {
-         @Override
-         protected void update() {
-            this.pendingThreads = HikariPool.this.getThreadsAwaitingConnection();
-            this.idleConnections = HikariPool.this.getIdleConnections();
-            this.totalConnections = HikariPool.this.getTotalConnections();
-            this.activeConnections = HikariPool.this.getActiveConnections();
-         }
-      };
-   }
-
-   // ***********************************************************************
-   //                      Non-anonymous Inner-classes
-   // ***********************************************************************
-
-   /**
-    * The house keeping task to retire idle connections.
-    */
-   private class HouseKeeper implements Runnable
-   {
-      private volatile long previous = clockSource.currentTime();
-
-      @Override
-      public void run()
-      {
-         // refresh timeouts in case they changed via MBean
-         connectionTimeout = config.getConnectionTimeout();
-         poolElf.setValidationTimeout(config.getValidationTimeout());
-         leakTask.updateLeakDetectionThreshold(config.getLeakDetectionThreshold());
-
-         final long now = clockSource.currentTime();
-         final long idleTimeout = config.getIdleTimeout();
-
-         // Detect retrograde time as well as forward leaps of unacceptable duration
-         if (now < previous || now > clockSource.plusMillis(previous, (2 * HOUSEKEEPING_PERIOD_MS))) {
-            LOGGER.warn("{} - Unusual system clock change detected, soft-evicting connections from pool.", poolName);
-            softEvictConnections();
-            fillPool();
-            return;
-         }
-
-         previous = now;
-
-         logPoolState("Before cleanup ");
-         for (PoolBagEntry bagEntry : connectionBag.values(STATE_NOT_IN_USE)) {
-            if (connectionBag.reserve(bagEntry)) {
-               if (bagEntry.evicted) {
-                  closeConnection(bagEntry, "(connection evicted)");
-               }
-               else if (idleTimeout > 0L && clockSource.elapsedMillis(bagEntry.lastAccess, now) > idleTimeout) {
-                  closeConnection(bagEntry, "(connection passed idleTimeout)");
-               }
-               else {
-                  connectionBag.unreserve(bagEntry);
-               }
-            }
-         }
-         
-         logPoolState("After cleanup ");
-
-         fillPool(); // Try to maintain minimum connections
-      }
-   }
-}
+/*

```

Comment:
Nice to have a pull request.  However, it seems like some setting, either eclipse editor setting or eclipse egit setting, messed with every line-ending.  So rather than a diff. of only changed lines, it is show that every line was deleted and added.  Fix the line-ending issue (preferably Unix-style) and commit again.  I think the pull request will automatically be updated.